tools/interop_matrix/testcases/csharp__v1.18.0

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
there actually need to be two files - csharp__v1.18.0  (which is a copy of the original csharp__master and csharpcoreclr__v1.18.0, which is a copy of old csharpcoreclr__master).

The current contents of the file are wrong (netcoreapp1.1 corresponds to CoreCLR called "csharpcoreclr" here, not to .NET desktop called "csharp" here)

See this snippet to understand how this works
https://github.com/grpc/grpc/blob/a3062ca04afef15327512d343cbd1768326b7500/tools/interop_matrix/run_interop_matrix_tests.py#L141